runtime/autoload/fswatch.vim

```diff
@@ -0,0 +1,43 @@
+if exists('g:loaded_watcher_provider')
+  finish
+endif
+
+let s:save_cpo = &cpo " save user coptions
+set cpo&vim " reset them to vim defaults
+
+command! -nargs=1 Watch call fswatch#watch_file(expand('<args>'))
+command! -nargs=1 Stop call fswatch#stop_watch(expand('<args>'))
+
+" function to prompt the user for a reload
+function! fswatch#PromptReload()
+  let choice = confirm("File changed. Would you like to reload?", "&Yes\n&No", 1)
+  if choice == 1
+    edit!
+  endif
+endfunction
+
+function! fswatch#PrintWatchers()
+  call luaeval("vim.fswatch.print_all()")
```

Comment:
Perhaps these would be better as `v:lua.vim.fswatch.print_all()` or similar. i don't know if we have a style preference for that or not.

Reply:
@teto suggested removing the vimscript wrappers all together and directly call the lua functions, I hadn't seen this review before now and so made a commit with the changes recommended by him.